.ci/spec/clean-packet.sh

```diff
@@ -4,18 +4,28 @@ csource="${BASH_SOURCE[0]}"
 while [ -h "$csource" ] ; do csource="$(readlink "$csource")"; done
 root="$( cd -P "$( dirname "$csource" )/../../" && pwd )"
 
-. "${root}/.ci/spec/env.sh"
 . "${root}/.ci/common.sh"
+. "${root}/.ci/spec/env.sh"
 
 pushd "${root}" > "${output}"
 
 echo "Cleaning up packet device..."
 
+unset PACKET_EXEC_PERSIST
+unset PACKET_EXEC_PRE_BUILTINS
+
+export PACKET_EXEC_REMOTE_DIRECTORY="run-${GITHUB_RUN_ID}"
```

Comment:
For some reason, by the time that the ci gets to this part the `job_id` changes from `run-${GITHUB_RUN_ID}` to `run1-${GITHUB_RUN_ID}`. This forces it back.